Rename shard_lengths to shard_sizes to be more inline with Tensor sizes.

pritamdamania87 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #66464

Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

pritamdamania committed:
Rename shard_lengths to shard_sizes to be more inline with Tensor sizes.

Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

bowangbj reviewed (approved):
Looks pretty good, thanks Pritam

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]

pritamdamania committed:
Update on "Rename shard_lengths to shard_sizes to be more inline with Tensor sizes."


Dimension sizes are referred to as `size` in general in PyTorch and
hence rename shard_lengths to shard_sizes.

#Closes: https://github.com/pytorch/pytorch/issues/65794

Differential Revision: [D31564153](https://our.internmc.facebook.com/intern/diff/D31564153/)

[ghstack-poisoned]
